apps/test-timer-pwm/main.cpp

```diff
@@ -0,0 +1,33 @@
+#include "HAL_PWM.h"
+#include "mbed.h"
+#include "stm32f4xx_hal.h"
+
+PwmOut pinA(PA_6);
+DigitalOut led(LED1);
+Serial pc(SERIAL_TX, SERIAL_RX, 9600);

```

Comment:
iirc @jetkov  had mentioned that printf actually works better than pc.printf. Is this right alex?